### ISSUE TITLE
Readme.md missing steps

### ISSUE THREAD

**Issue by lampdev** [genyaevgeney/work-event-project#20]
1. copying .env.example to .env, otherwise getting `  ErrorException  : file_get_contents(/Users/od396a/work-event-project/.env): failed to open stream: No such file or directory

  at /Users/od396a/work-event-project/vendor/laravel/framework/src/Illuminate/Foundation/Console/KeyGenerateCommand.php:96`

2. no DB connection is configured on migrate and seed steps. Either use sqlite and provide the .env setup instructions or docker environment for mysql

3. provide instructions for known issue of key length:

work-event-project|master ⇒ php artisan migrate
Migration table created successfully.
Migrating: 2020_01_23_161350_create_articles_table

   Illuminate\Database\QueryException  : SQLSTATE[42000]: Syntax error or access violation: 1071 Specified key was too long; max key length is 767 bytes (SQL: alter table `articles` add unique `articles_title_unique`(`title`))

4. change the order of build , lint and serve, so that serve is the latest step

5. unit tests are failing on clean install:

work-event-project|master ⇒ vendor/bin/phpunit
PHPUnit 8.5.2 by Sebastian Bergmann and contributors.

E                                                                   1 / 1 (100%)

Time: 281 ms, Memory: 18.00 MB

There was 1 error:

1) Tests\Feature\ControllerTest::testArticleController
ErrorException: Undefined index: data

/Users/od396a/work-event-project/tests/Feature/ControllerTest.php:29

ERRORS!
Tests: 1, Assertions: 0, Errors: 1.

6. add a brief project description as a separate first section of Readme

**Comment by aderiyenko** [comment]
I see the issues are being closed, but see no PR's / new branches / commits in history. Can we do one issue <-> one PR? 
1. Reference the issue in the PR description. 
2. add unit and (possibly) an integration test for the fix
3. close only after the PR is merged

**Comment by lampdev** [comment]
I see the issues are being closed, but see no PR's / new branches / commits in history. Can we do one issue <-> one PR?

Reference the issue in the PR description.
add unit and (possibly) an integration test for the fix
close only after the PR is merged

**Comment by genyaevgeney** [comment]
I fixed all issues except one locally and wanted to push all changes. I will make a separate pull request for each issues as you wrote.
